src/main/java/reportservice/ReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package reportservice;
+
+import db.Storage;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    @Override
+    public String createReport() {
+        StringBuilder builderForGetContentFromMap = new StringBuilder();
+        for (Map.Entry<String, Integer> entry: Storage.getStorage().entrySet()) {
+            builderForGetContentFromMap.append(entry.getKey()).append(",")
+                    .append(entry.getValue()).append(",");
```

Comment:
append new line here and you won't need to split result below
and use System.lineSeparator() for new line symbol